"Invalid value undefined for setWidth" in standard-version

This yoga error occurs when a command using Ink v3 happens in husky + [standard-version](https://github.com/conventional-changelog/standard-version). I would assume this happening because the hook runs in an environment where there is no terminal width to read.

Perhaps there should be a check/default on the `terminalWidth` argument here?
https://github.com/vadimdemedes/ink/blob/155e1a042b12e82929bdb26571385b5dea0f0b39/src/renderer.ts#L12-L13

```
Error: Invalid value undefined for setWidth
    at Bound.<anonymous> (/Users/jared/Code/oss/lundle/node_modules/yoga-layout-prebuilt/yoga-layout/dist/entry-common.js:167:17)
    at Bound.prototype.<computed> [as setWidth] (/Users/jared/Code/oss/lundle/node_modules/yoga-layout-prebuilt/yoga-layout/dist/entry-common.js:133:22)
    at Object.exports.default (/Users/jared/Code/oss/lundle/node_modules/ink/build/renderer.js:11:19)
    at Ink.onRender (/Users/jared/Code/oss/lundle/node_modules/ink/build/ink.js:49:78)
    at Ink.unmount (/Users/jared/Code/oss/lundle/node_modules/ink/build/ink.js:172:14)
    at EventEmitter.emit (events.js:315:20)
    at emit (/Users/jared/Code/oss/lundle/node_modules/signal-exit/index.js:78:11)
    at processEmit [as emit] (/Users/jared/Code/oss/lundle/node_modules/signal-exit/index.js:156:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 7
npm ERR! lundle@0.4.8 check-types: `FORCE_COLOR=1 node dist/main/bin check-types`
npm ERR! Exit status 7
npm ERR! 
npm ERR! Failed at the lundle@0.4.8 check-types script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jared/.npm/_logs/2020-07-05T21_38_36_580Z-debug.log
husky > pre-commit hook failed (add --no-verify to bypass)
```

--- a/src/ink.tsx
+++ b/src/ink.tsx
@@ -115,7 +115,9 @@ export default class Ink {
 
 		const {output, outputHeight, staticOutput} = render(
 			this.rootNode,
-			this.options.stdout.columns
+			// The 'columns' property can be undefined when not using a TTY.
+			// In that case we fall back to 80.
+			this.options.stdout.columns ?? 80
 		);
 
 		// If <Static> output isn't empty, it means new children have been added to it

